tests/renderServer.spec.ts

```diff
@@ -60,4 +61,27 @@ describe('renderServer', () => {
       expect(resp.text).toBe('"style" must be [dayChart]');
     });
   });
+
+  describe('GET /health-check', () => {
+    it('Responds 200 OK when configured', async () => {
+      const config = new ConfigService('./example');
+      config.setVersion('1.0-test');
+
+      const app = renderServer(config);
+      const resp = await supertest(app).get('/health-check').send();
+
+      expect(resp.status).toBe(200);
+      expect(resp.text).toBe('OK');
+    });
+
+    it("503's before the config is loaded", async () => {
```

Comment:
I think I see the issue here now: we _assume_ this can only happen when a config is not set but it would happen when a config is lacking a version, right?

Reply:
The config validation will ensure the version is set

Reply:
So we refuse to load a config without a version then? In that case, yeah, the only time this can happen is before _any_ config is set as afterwards the system would reject configs without versions, right?

Reply:
Correct.

I am working on the PR right now that polls for new configurations.